packages/classic/scss/floating-label/_variables.scss

```diff
@@ -34,7 +34,7 @@ $kendo-floating-label-focus-offset-y: 0 !default;
 
 /// The transition of the Floating Label.
 /// @group floating-label
-$kendo-floating-label-transition: .2s ease-out !default;
+$kendo-floating-label-transition: k-motion(short-4, ease-out, transform, color, top, inset-inline-start) !default;
```

Comment:
Maybe here should be animated all props?

Reply:
The same point as with Bootstrap, there is no need to transition all properties.